one_click/gpu_provisioning_playbook.yaml

```diff
@@ -0,0 +1,192 @@
+            k8s_info:
+              kind: Deployment
+              wait: yes
+              name: nfd-controller-manager
+              namespace: openshift-nfd
+              wait_condition:
+                type: Available
+                status: 'True'
+                reason: MinimumReplicasAvailable
+      - name: KMM - Install Kernel Module Management Operator
+        tags: 
+          - kmm
+        block: 
+        - name: KMM - Create openshift-kmm namespace
+          k8s:
+            name: openshift-kmm
+            api_version: v1
+            kind: Namespace
+            state: present
+            wait: yes
+        - name: KMM - Create OperatorGroup v1 in openshift-kmm namespace
+          k8s:
+            definition:
+              apiVersion: operators.coreos.com/v1
+              kind: OperatorGroup
+              metadata:
+                name: kernel-module-management
+                namespace: openshift-kmm
+            wait: yes
+        - name: KMM - Create Subscription for KMM Operator
+          k8s:
+            definition:
+              apiVersion: operators.coreos.com/v1alpha1
+              kind: Subscription
+              metadata:
+                name: kernel-module-management
+                namespace: openshift-kmm
+              spec:
+                channel: stable
+                installPlanApproval: Automatic
+                name: kernel-module-management
+                source: redhat-operators
+                sourceNamespace: openshift-marketplace
+                startingCSV: kernel-module-management.v2.1.1
+            wait: yes
+            wait_condition:
+              reason: AllCatalogSourcesHealthy
+              type: CatalogSourcesUnhealthy
+              status: 'False'
+        - name: KMM - Wait until the kmm-operator-controller Deployment is available
+          k8s_info:
+            kind: Deployment
+            wait: yes
+            name: kmm-operator-controller
+            namespace: openshift-kmm
+            wait_condition:
+              type: Available
+              status: 'True'
+              reason: MinimumReplicasAvailable
+        - name: KMM - Update KMM CM to set FW path 
+          command: |
+            oc patch configmap kmm-operator-manager-config -n openshift-kmm --type='json' -p='[{"op": "add", "path": "/data/controller_config.yaml", "value": "healthProbeBindAddress: :8081\nmetricsBindAddress: 127.0.0.1:8080\nleaderElection:\n  enabled: true\n  resourceID: kmm.sigs.x-k8s.io\nwebhook:\n  disableHTTP2: true\n  port: 9443\nworker:\n  runAsUser: 0\n  seLinuxType: spc_t\n  setFirmwareClassPath: /var/lib/firmware"}]'
+        - name: KMM - Delete the KMM operator controller pod for `ConfigMap` changes to take effect.
+          shell: oc get pods -n openshift-kmm | grep -i "kmm-operator-controller-" | awk '{print $1}' | xargs oc delete pod -n openshift-kmm
+        - name: KMM - wait 10 seconds until KMM operator pod is up and running
```

Comment:
It isn't the right way for KMM to set up the FW path, that right way should be set from the CRD. For this prototype implementation, I can accept it. But please file an issue for KMM Operator and work with the KMM owner on the right solution.